src/modules/sync/utils/fetchers.ts

```diff
@@ -97,3 +97,9 @@ export const fetchAssetFolders = (client: ManagementClient) =>
     .listAssetFolders()
     .toPromise()
     .then(res => res.rawData.folders);
+
+export const fetchCollections = (client: ManagementClient) =>
+  client
+    .listCollections()
+    .toPromise()
+    .then(res => res.data.collections.map(c => c._raw));
```

Comment:
You can just get `res.rawData.collections` and avoid the `map`. It's not super important, but the raw array is already there, so why not use it?